### PR TITLE
Integrate rootstrap-ui and remove foundation

### PR DESCRIPTION
**Pull request by bernabe9**

### CODE REVIEW

**Comment by peter7z** [comment]
I didn't even know we had this foundation thing.

File: README.md [hunk 1]
```diff
@@ -91,6 +91,7 @@ The server is prepared to fetch data directly from the backend before rendering
 | [Redux Persist](https://github.com/rt2zz/redux-persist) | Persist and rehydrate your redux store |
 | [Immer](https://github.com/immerjs/immer) | Allows you to work with immutable state in a more convenient way. |
 | [React Intl](https://github.com/yahoo/react-intl/) | Localization for language support. |
+| [Rootstrap UI](https://github.com/yahoo/react-intl/) | Rootstrap UI is the components & styles library to build user interfaces |
```
Comment (fsarachu) [inline README.md:94]:
```suggestion
| [Rootstrap UI](https://github.com/rootstrap/rootstrap-ui/) | Rootstrap UI is the components & styles library to build user interfaces |
```

Comment (bernabe9) [reply README.md:94]: Good catch 😅

**Comment by peter7z** [comment]
I hope you are squashing after that last commit...

**Comment by bernabe9** [comment]
@peter7z we shouldn't squash commits in public branches (it's a destructive action)... Instead, I usually "Squash and merge" to `master` to keep the git history clean :)

**Pull request merged**